docs/user-guides/configuration.md

```diff
@@ -223,24 +250,25 @@ gateway:
 
 #### Index replica
 
-`gateway.lb.gateway_config.index_replica` represents how many agent pods that a vector will be inserted into.
+`gateway.lb.gateway_config.index_replica` represents how many Vald Agent pods a vector will be inserted into.
+We recommend setting it as one-third of the number of Vald Agent pods.
```

Comment:
I think we didn't recommend to set it to one-third.
e.g. User deployed 30 agents, in this sentence it means we recommended to set the index replica to 10.

```suggestion
We recommend the maximum value of the index replica is one-third of the Vald agent deployed.
```

Reply:
```suggestion
The maximum value of the index replica should be one-third of the Vald Agent pods deployed.
```

Reply:
@kevindiu how about it?

Reply:
@vankichi LGTM